docs/topics/assembly_developing-and-deploying-spring-reactive-application.adoc

```diff
@@ -0,0 +1,25 @@
+[id='developing-reactive-spring-boot-applications-using-spring-boot-vertx_{context}']
+= Developing reactive applications using {SpringReactive}
+
+:parent-context: {context}
+
+include::con_spring-reactive-introduction.adoc[leveloffset=+1]
+
+//include::con_spring-web-reactive.adoc
+include::proc_creating-a-reactive-http-service-with-webflux.adoc[leveloffset=+1]
+
+include::proc_using-basic-authentication-in-http-service-webflux.adoc[leveloffset=+1]
+
+include::proc_using-oauth2-with-reactive-http-web-service-webflux.adoc[leveloffset=+1]
+
+include::proc_creating-reactive-smtp-email-app.adoc[leveloffset=+1]
+
+//concept module added
+include::con_server-sent-events.adoc[leveloffset=+1]
+include::proc_using-server-sent-events-spring-reactive-app.adoc[leveloffset=+1]
+
+//In addition to xref:example-rest-http-spring-boot[using an example], you can create new {runtime} applications from scratch and deploy them to OpenShift.
```

Comment:
Please remove comments.

Unaddressed TODOs?

Reply:
TODOs addressed and/or removed, respectively